src/webresource-handler/handlers/NoopHandler.ts

```diff
@@ -18,4 +26,43 @@ export class NoopHandler implements WebResourceHandler {
 	public async onPreRespond(webResource: WebResource): Promise<WebResource> {
 		return webResource;
 	}
+
```

Comment:
I really dislike this file as a whole, I am considering just dropping it as then treating all the cases where webResourceHandler is undefined properly

Reply:
What is the purpose of the noop handler?

Reply:
The sole purpose of this file is this line: https://github.com/balena-io/pinejs/blob/master/src/webresource-handler/handlers/NoopHandler.ts#L7

We must resume the file stream in order for the request to be able to move on. I am considering just dropping this thing and then checking if webResource is null, manually call the `.resume()`